src/Dmishh/Bundle/SettingsBundle/Controller/SettingsController.php

```diff
@@ -107,4 +110,20 @@ protected function manage(Request $request, SettingsOwnerInterface $owner = null
             )
         );
     }
+
+    /**
+     * Get SecurityContext service
+     * @param string $service The service name
+     *
+     * @return mixed The service
+     */
+    private function getSecurityContext($service)
```

Comment:
Also, `security.authorization_checker` is a different service, so it shouldn't be involved into getting security context